### ISSUE TITLE
Download button should be shown by default

### ISSUE THREAD

**Issue by aviaryan** [aashutoshrathi/Insta-Downloader-Extension#10]
Would it be possible to show the "Download" button by default when an Instagram page loads? 
This will certainly save some hassle.

**Comment by aashutoshrathi** [comment]
Actually, reading about it and also thinking that on what basis should we do that, as it only grabs posts at a time. 😅

**Comment by dufferzafar** [comment]
Yeah, clicking on a toolbar button only to have it display a Download button is a bit odd.

**Comment by aashutoshrathi** [comment]
Fixed this @aviaryan.
Thanks for pointing out.
If possible please test again.

**Comment by aviaryan** [comment]
@aashutoshrathi You should update the GIF as well.

**Comment by aashutoshrathi** [comment]
Oh Yeah! :sweat_smile:

**Comment by aashutoshrathi** [comment]
@aviaryan  updated it with the latest demo, check out once. All features added. 🎉
